README.md

```diff
@@ -1,2 +1,133 @@
 # components-java-sdk
-Dapr SDK to compose pluggable components in Java.
+
+[Dapr] SDK to compose Pluggable components in Java. Also defines some sample components.
+
+For further information on Pluggable Components, check:
+*  [(Proposal) (Updated) gRPC Components (aka "Pluggable components") · Issue #4925](https://github.com/dapr/dapr/issues/4925)
+*  [Pluggable Components QuickStart]
+
+# Getting started
+
+## Pre-Requisites
+
+We build on [Dapr Java SDK's dependencies](https://github.com/dapr/java-sdk/blob/master/README.md#getting-started) and
+additionally require the following:
+
+* [Gradle 7.x](https://gradle.org/install/)
+* [Docker](https://docs.docker.com/engine/install/)
+
+## Developing new components
+
+The code under `src/main/java/org/example/` exemplifies how one can
```

Comment:
nit:

I noticed that for [dapr java-sdk](https://github.com/dapr/java-sdk) we kept the examples folder at repository root level, should we do the same?

Reply:
For the time being I don't want to do that. In a later PR I will split SDK and example code apart but at this first PR I just want to lay the basis SDK code. Moving examples to their own separate directory seems like the right thing to do. I cut issue #8 keep track of this.